### PR TITLE
staticassert.d: fix error messages to use syntax highlighting

### PR DESCRIPTION
**Pull request by WalterBright**

### CODE REVIEW

File: test/fail_compilation/ctfe14465.d [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
+        super(msg);
+    }
+}
+
+bool foo()
+{
+    throw new E("message");
+}
+
+static assert(foo());
```
Comment (CyberShadow) [inline test/fail_compilation/ctfe14465.d]: What happened here? Are the line endings correct?

Comment (CyberShadow) [inline test/fail_compilation/ctfe14465.d]: Looks like they were CRLF and now they are LF (as they should have been).

**Review by CyberShadow — APPROVED** [review]

File: test/fail_compilation/ctfe14465.d (hunk 1 above)
Comment (WalterBright) [inline test/fail_compilation/ctfe14465.d]: I didn't realize that change happened, but part of my submission process is to convert to LF line endings.

Comment (CyberShadow) [inline test/fail_compilation/ctfe14465.d]: PR to fix the rest of them: https://github.com/dlang/dmd/pull/6793

**Comment by CyberShadow** [comment]
Please fix conflicts

**Pull request merged**